contracts/seed/Seed.sol

```diff
@@ -43,6 +43,7 @@ contract Seed {
     IERC20 public seedToken;
     IERC20 public fundingToken;
     uint256 public fee; // Success fee expressed as a % (e.g. 10**18 = 100% fee, 10**16 = 1%)
+    uint256 public maxFee = 45 / 100 *10**18; // Max fee expressed as a % (e.g. 45 / 100 * 10**18 = 45% fee) 
```

Comment:
it should be a constant defined in all CAPS LOCK, like `PRECISION`, and to be defined near `PRECISION` constant

Reply:
```suggestion
    uint256 internal constant MAX_FEE = 45 / 100 * 10**18; // Max fee expressed as a % (e.g. 45 / 100 * 10**18 = 45% fee) 
```